docs/developers/emp-deployment.md

```diff
@@ -3,7 +3,7 @@ title: Deploying an EMP Contract
 sidebar_label: Deploying an EMP Contract
 ---
 
-This tutorial will show you how to deploy an ExpiringMultiParty (EMP) contract by using the UMA `launch-emp` repository. By following these instructions, you will be able to launch a custom EMP on the Kovan testnet or Ethereum mainnet.
+This tutorial will show you how to deploy an ExpiringMultiParty (EMP) contract by using the UMA `launch-emp` repository. By following these instructions, you will be able to launch a custom EMP on the Kovan testnet or Ethereum mainnet. The following tutorial is highly recommended for you to get familiar with the process. If you are familiar with process and understand how to correctly set parameters you can use a community made contract [deployer tool](https://syntheticbuilder.xyz/) to launch your EMP. 
```

Comment:
Can we present the syntheticbuilder as more of an alternative?

Just separating this out into a different paragraph after "...Kovan testnet or Ethereum mainnet" and saying something like:

"As an alternative deployment option, you can also launch an EMP through this community built synthetic token builder. This provides an easier deployment process, but may limit some customization. As a disclaimer, this tool has been used successfully by contract deployers, but is community built and maintained, so please use at your own risk."

Thoughts?